### ISSUE TITLE
Separate job-related schemas into their own package

### ISSUE THREAD

**Issue by javiermtorres** [mozilla-ai/lumigator#415]
**Is your feature request related to a problem? Please describe.**
In the current architecture, schemas needed for a job (like schemas defining the input config for a job) are currently defined within the job folder. However, they should be used within lumigator to format the job config command line params instead of using templates.

**Describe the solution you'd like**
A separate package (as `sdk` and `schemas` are available right now) holding job-specific schemas. When a new job would be added, the lumigator-specific packages would not need to be updated, only the job-specific config package.

**Describe alternatives you've considered**
Dynamic import (`importlib`) of job-related schemas directly into the backend and sdk.

**Additional context**
N/A

**Comment by njbrake** [comment]
@javiermtorres Can this be closed, since we merged https://github.com/mozilla-ai/lumigator/issues/670?

**Comment by javiermtorres** [comment]
@njbrake yes, the issue itself is solved and the situation is actually somewhat different, so it makes no sense to keep it open. Sorry for not having noticed this issue earlier.
